duet/views/mobile_ios_country.view.lkml

```diff
@@ -0,0 +1,368 @@
+        country,
+        max(ios_store_updated) AS ios_store_updated,
+        max(latest_date) AS latest_date,
+        sum(product_page_views) AS product_page_views,
+        sum(first_time_installs) AS first_time_installs,
+        sum(installations_opt_in) AS installations_opt_in,
+        sum(first_seen) AS first_seen,
+        sum(activated) AS activated
+      FROM
+        apple_country_metrics
+      JOIN
+        last_seen
+      USING
+        (submission_date, country)
+      CROSS JOIN
+        period
+      WHERE
+        submission_date
+        BETWEEN start_date
+        AND end_date
+      GROUP BY
+        1,
+        2
+      ORDER BY
+        1,
+        2
+       ;;
+  }
+
+  parameter: app_id {
+    description: "The name of the application in the `org.mozilla` namespace."
+    type:  unquoted
+    default_value: "firefox"
+    allowed_value: {
+      label: "Firefox"
+      value: "firefox"
+    }
+    allowed_value: {
+      label: "Focus"
+      value: "focus"
+    }
+    allowed_value: {
+      label: "Klar (Focus)"
+      value: "klar"
+    }
+  }
+
+  # Choose how far back in history to look
+  parameter: history_days {
+    description: "The number of days to include in the aggregation period."
+    type:  number
+    default_value: "7"
+    allowed_value: {
+      value: "1"
+    }
+    allowed_value: {
+      value: "7"
+    }
+    allowed_value: {
+      value: "28"
+    }
+    allowed_value: {
+      value: "84"
+    }
+  }
+
+  parameter: period_offset {
+    description: "An offset into the aggregation period, used to calculate percent difference since last period."
+    type: number
+    default_value: "0"
+    allowed_value: {
+      value: "0"
+    }
+    allowed_value: {
+      value: "1"
+    }
+  }
+
+  dimension: bucket {
+    description: "The geographical bucket that the lies under. Do not combine with country unless this is set to overall."
+    type: "string"
+    hidden: yes
+    suggest_explore: country_buckets
+    suggest_dimension: country_buckets.bucket
+  }
+
+  dimension: join_field {
+    type: yesno
+    description: "Always set to true. Allows to merge explores."
+    sql: TRUE ;;
+  }
+
+  dimension_group: submission {
+    description: "The submission date of the data."
+    type: time
+    datatype: date
+    timeframes: [
+      raw,
+      time,
+      date,
+      week,
+      month,
+      quarter,
+      year
+    ]
+    sql: ${TABLE}.submission_date ;;
+  }
+
+  dimension: country {
+    description: "The country code for the aggregates"
+    type: string
+    sql: ${TABLE}.country ;;
+  }
+
+  dimension: ios_store_updated {
+    description: "The date of the last apple store import"
+    type: date
+    datatype: date
+    sql: ${TABLE}.ios_store_updated ;;
+  }
+
+  dimension: latest_date {
+    description: "The most recent submission date used for analysis"
+    type: date
+    datatype: date
+    sql: ${TABLE}.latest_date ;;
+  }
+
+  measure: product_page_views {
```

Comment:
What are your thoughts on putting `step_N` before each of these? That would make it clearer what the funnel actually is in the explore, and would then also order the measures by their step count (rather than alpha by name).

Reply:
It might not be a bad idea, though the name of the measure will get somewhat long. This would probably go in a follow up so the android view is consistent in naming.